### DISCUSSION TITLE
Black window and no modules

### DISCUSSION THREAD

**Discussion by gambansumbl** [DISTRHO/Cardinal#97] [General]
Hi and thank you for your work on Cardinal!

I am trying to run the vst2 version (build #465) in FL studio on Windows, but when I load the plugin, there is only black window with an upper menu and scrollbars. There are also no modules to add. Maybe the plugin doesn't see the "resources" folder. I have this folder in the same directory as the plugin dll file. Is this correct? Am I missing something? I would appreciate some help.

Cheers!

**Comment by falkTX** [comment]
I thought this was fixed but guess not.
What is most likely happening here is that you moved the contents of the CardinalSynth.vst somewhere else, leaving the CardinalSynth.vst folder behind.
This can sometimes fail.

So instead move the CardinalSynth.vst folder itself, not its contents, to the VST plugin path.

**Reply by gambansumbl** [reply]
Yes! This was it. It works now.
Thanks!

**Reply by Detoxgit** [reply]
at first i did move the contents along with the vst2 folder into my vst folder,then it showed black screen then i again reinstalled it and tried opening it i am getting the same black screen
